### DISCUSSION TITLE
Python requirement should be `>=3.7`

### DISCUSSION THREAD

**Discussion by equal-l2** [fcakyon/yolov5-pip#202] [General]
`yolov5-pip` currently requires python with `>=3.6`, but `huggingface-hub>=0.11.1`, a dependency of that, requires python `>=3.7.0`, so Python 3.6 is not supported anyway.
(I should make a pull request, but I'm not sure about python packaging so no clue where I should modify)

**Comment by fcakyon** [comment]
Python3.6 end of life reached 1 year ago. You should upgrade your Python: https://endoflife.date/python

**Comment by equal-l2** [comment]
I'm sorry but I'm not using Python 3.6 .

What I intend to say is that YOUR library says it supports Python 3.6, which is not true, and you should amend that requirement.

**Reply by fcakyon** [reply]
You are right, this PR will be fixing it: https://github.com/fcakyon/yolov5-pip/pull/203

**Reply by equal-l2** [reply]
Thank you!
